config/environments/development.rb

```diff
@@ -56,9 +56,21 @@
   config.active_record.verbose_query_logs = true
 
   # Mailcatcher configuration
+  # config.action_mailer.delivery_method = :smtp
+  # config.action_mailer.smtp_settings = { :address => '127.0.0.1', :port => 1025 }
+  # config.action_mailer.raise_delivery_errors = false
+
+  # Action Mail configurations
   config.action_mailer.delivery_method = :smtp
-  config.action_mailer.smtp_settings = { :address => '127.0.0.1', :port => 1025 }
-  config.action_mailer.raise_delivery_errors = false
+  # config.action_mailer.default_url_options = { :host => 'subvisuwheel.netlify.app', :protocol => 'https' }
+  config.action_mailer.smtp_settings = {
+    :address => 'smtp.gmail.com',
+    :port => 587,
+    :user_name => ENV["USER_NAME"],
+    :password => ENV["USER_PASSWORD"],
+    :authentication => 'plain',
+    :enable_starttls_auto => true
+  }
```

Comment:
Why are you removing the configuration for using Mailcatcher, and sending real emails? This is development, you can make some tests with the real thing, but it doesn't seem a good thing to have it every time. Mailcatcher should be enough for what you need

Reply:
I was just testing if the Action Mailer was actually sending emails, I'll remove that and just leave the Mailcatcher